infrastructure/k8s/bin/helmit.py

```diff
@@ -75,13 +80,28 @@ def render_cronjob(project_name, secret_keys, deployment):
     context.update(deployment)
     format_values(context)
     template = yaml.load(open("templates/cron-job.yaml"), Loader=yaml.SafeLoader)
-    print("---")
-    print(yaml.dump(jsone.render(template, context)))
-
+    # print("---")
+    # print(yaml.dump(jsone.render(template, context)))
+    write_file(template, context, "cron")
+
+def write_file(template, context, prefix):
+    filename = chartsdir + "/taskcluster-" + prefix + "-" + p.split("/")[-1:][0]
+    try:
+        f = open(filename, "a+")
+        f.write(yaml.dump(jsone.render(template, context)))
+        f.close()
+    except: 
+        print("failed to write "+ filename)
```

Comment:
Instead of building strings this way, use an f-string. See https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python for an intro.